docs/Community/Committers.md

```diff
@@ -0,0 +1,65 @@
+Project Committers
+==================
+
+List of Committers
+------------------
+
+The following people have been granted commit permissions (that is, write
+access) to the Open Enclave SDK by the Community Governance Committee. The area
+column describes which technical areas each committer is most interested in, and
+therefore should usually be consulted for changes relating to that area.
+
+
+| Name                  | GitHub Alias        | Area                           |
+|-----------------------|---------------------|--------------------------------|
+| Amaury Chamayou       | achamayou           | Build, CCF Integration         |
+| Anand Krishnamoorthi  | anakrish            | Debugging, SGX, EDL, Dev Tools |
+| Andrew Schwartzmeyer  | andschwa            | EDL, CMake, Dev Tools, Website |
+| Brett McLaren         | BRMcLaren           | Build, CMake, CI               |
+| Brian Telfer          | Britel              | TrustZone, Attestation         |
+| Simon Leet            | CodeMonkeyLeet      | SGX, APIs                      |
+| Dave Thaler           | dthaler             | TrustZone, APIs, Dev Tools     |
+| Emil Alexandru Stoica | EmilAlexandruStoica | Build, CMake, CI, Ansible      |
+| Akash Gupta           | gupta-ak            | SGX, Attestation, APIs         |
+| Hernan Gatta          | HernanGatta         | TrustZone                      |
+| Sergio Wong           | jazzybluesea        | Attestation, SGX               |
+| Jiri Appl             | jiria               | Attestation, TrustZone         |
+| John Kordich          | johnkord            | Build, CI, Dev Tools, Release  |
+| Jordan Hand           | jhand2              | Windows, Build, SGX            |
+| Xuejun Yang           | jxyang              | SGX                            |
+| Mike Brasher          | mikbras             | SGX, APIs, EDL                 |
+| Marius Oprin          | oprinmarius         | Build, CMake, CI, Ansible      |
+| Paul Allen            | paulcallen          | TrustZone                      |
+| Radhika Jandhyala     | radhikaj            | SGX, APIs, Website             |
+| Shruti Ratnam         | shruti25ratnam      | Attestation                    |
+| Cheng-mean Liu        | soccerGB            | Attestation, SGX               |
+| Bruce Campbell        | yakman2020          | Windows, SGX                   |
+
+Current Expectations
+--------------------
+
+However, it is up to each committer to determine who should review which PR, and
```

Comment:
I am wondering if I as a committer open a PR, can I decide that only committer A and committer B review it? If committer C reviews it despite not being assigned, can C request changes? I understand that this is probably what was in the old docs, so I am just leaving this in so I understand the answer and dont ask for this to be addressed in the current PR.

Reply:
Yeah, that's what was in the old doc, and this situation crossed my mind as well.

I would propose that approval from at least one SIG lead other than the authori is required to merge code, and approval from two or more is preferred (especially for larger changes)